documentation/book/proc-configuring-internal-clients-to-trust-cluster-ca.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='configuring-internal-clients-to-trust-cluster-ca-{context}']
+= Configuring internal clients to trust the cluster CA 
+
+This procedure will show you how to configure a Kafka client that is inside the {ProductPlatformName} cluster (that is, connecting to the `tls` listener on port 9093) so that it trusts the cluster CA certificate.
```

Comment:
_Please reword as follows:_

This procedure describes how to configure a Kafka client that resides inside the {ProductPlatformName} cluster—connecting to the `tls` listener on port 9093 —to trust the cluster CA certificate.